src/libm/CMakeLists.txt

```diff
@@ -0,0 +1,59 @@
+# Helper executable: generates parts of the sleef header file
+add_executable(${TARGET_MKRENAME} mkrename.c)
+target_compile_options(${TARGET_MKRENAME}
+  PRIVATE ${FLAGS_WALL})
+
+# TODO: set the parameters for mkrename run for the SIMD implementation
+set(MKRENAME_PARAMS "")
+
+configure_file(
+  ${CMAKE_CURRENT_SOURCE_DIR}/GenerateSleefHeader.cmake.in
+  ${CMAKE_CURRENT_BINARY_DIR}/GenerateSleefHeader.cmake @ONLY)
+
+# Add custom command to generate the sleef header at build time
+add_custom_command(
+  OUTPUT ${CMAKE_BINARY_DIR}/include/sleef.h
```

Comment:
Please define a target in the top level folder for `${CMAKE_BINARY_DIR}/include/sleef.h`.